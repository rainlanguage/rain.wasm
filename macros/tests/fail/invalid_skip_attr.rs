#[macro_use]
extern crate wasm_bindgen_utils_macros;

struct TestStruct;

#[wasm_export]
impl TestStruct {
    #[wasm_export(skip = "string")]
    pub async fn some_static_method(arg: String) -> Result<String, Error> {
        Ok(String::new())
    }
}

fn main() {}
