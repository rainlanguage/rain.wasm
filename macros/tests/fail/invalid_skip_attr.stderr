error: unexpected token in attribute
 --> tests/fail/invalid_skip_attr.rs:8:24
  |
8 |     #[wasm_export(skip = "string")]
  |                        ^
